src/main/java/core/basesyntax/FruitShopHW.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Activity;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.activity.strategy.ActivityHandler;
+import core.basesyntax.service.activity.strategy.BalanceHandler;
+import core.basesyntax.service.activity.strategy.PurchaseHandler;
+import core.basesyntax.service.activity.strategy.ReturnHandler;
+import core.basesyntax.service.activity.strategy.SupplyHandler;
+import core.basesyntax.service.impl.ActivitiesProcessorImpl;
+import core.basesyntax.service.impl.ActivityStrategyImpl;
+import core.basesyntax.service.impl.CsvFileReader;
+import core.basesyntax.service.impl.CsvFileWriter;
+import core.basesyntax.service.impl.DataParserImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopHW {
+    private static final String FILE_NAME = "src/main/resources/activities.csv";
+    private static final String REPORT_NAME = "src/main/resources/report.csv";
+    private static final List<Activity> LIST_OF_ACTIVITIES = new ArrayList<>();
```

Comment:
You don't need constants for services and list of activities, declare them in `main()`